harness/config/external-images.yml

```
@@ -19,20 +19,28 @@ function('external_images', [services]): |
   = join(' ', $externalImages);
 
 
-command('external-images config'):
+command('external-images config --skip-exists'):
```

```suggestion
command('external-images config [--skip-exists]'):
```
To fix issues running locally